cmd/jimmsrv/service/register.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (\d{4}) Canonical.
```

Comment:
Just realized jimm still uses the regex for goheader which is buggy. I'll make a PR to use the same built-in value as in the LP repos but you'll need to look out for this before v3 is merged into this feature branch.